recipe/meta.yaml

```diff
@@ -13,24 +13,24 @@ source:
 
 build:
   skip: true  # [win or py3k]
-  number: 3
+  number: 4
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - toolchain
-    - boost 1.64.*
+    - boost 1.65.1
     - python
     - setuptools >=18.0
-    - numpy x.x
+    - numpy 1.11.*
     - numpy >=1.7
```

Comment:
I'm guessing we can try an older `numpy` version, but this remains compatible with the versions already served by this recipe. One can easily submit a follow-up PR to relax this further.